### PR TITLE
Update people.md

### PR DESCRIPTION
**Pull request by gamedaygeorge**
Updates to SC: updating Rob's Chair Term, updates to reflect FrontierSI joining SC.

### CODE REVIEW

File: people.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Names are followed by GitHub usernames.
 - Tony Butzer, @tonybutzer ([USGS](https://www.usgs.gov/))
 - Syed Rizvi, @srrizvi ([AMA](http://www.ama-inc.com/))
 - Andrew Cherry, @AMA-AC ([AMA](http://www.ama-inc.com/))
-- Chris Morgan, @chriscmorgan (Secretariat member) (FrontierSI)
+- Chris Morgan, @chriscmorgan (FrontierSI)
+- Caitlin Adams (FrontierSI)
```
Comment (alexgleith) [inline people.md:19]:
```suggestion
- Caitlin Adams, @caitlinadams (FrontierSI)
```

**Review by alexgleith — COMMENTED** [review]
Added Caitlin's github username

**Comment by gamedaygeorge** [comment]
@woodcockr this one as well on SC membership. Just to be crossing t's and dotting i's.

**Review by woodcockr — APPROVED** [review]
All good

**Pull request merged**